pyproject.toml

```diff
@@ -11,7 +11,8 @@ include = ["CHANGELOG.md"]
 [tool.poetry.dependencies]
 python = "^3.7.1"
 thrift = "^0.13.0"
-pandas = "^1.3.0"
+pandas = "^1.3.5"
+numpy = [{version="^1.23.2", python=">=3.8"}, {version="^1.21.6", python="<3.8"}]
```

Comment:
I am not sure I have done what I want here. I want to specify "use at least this version of numpy for python >= 3.8". But this ends up as 

```{':python_version < "3.8"': ['numpy>=1.21.6,<2.0.0'],
 ':python_version >= "3.8"': ['numpy>=1.23.2,<2.0.0']}
```

Which is likely fine but pandas 1.3.5 for example ends up with

```install_requires = 
        numpy>=1.17.3; platform_machine!='aarch64' and platform_machine!='arm64' and python_version<'3.10'
        numpy>=1.19.2; platform_machine=='aarch64' and python_version<'3.10'
        numpy>=1.20.0; platform_machine=='arm64' and python_version<'3.10'
        numpy>=1.21.0; python_version>='3.10'
```

Reply:
This can be addressed by using `>=$version`, but I still feel that obtaining numpy version information via pandas is better.